ide/api/phone.py

```diff
@@ -22,14 +23,18 @@ def list_phones(request):
         params={'client_id': settings.SOCIAL_AUTH_PEBBLE_KEY})
 
     if response.status_code != 200:
-        return json_failure(response.reason)
+        if 400 <= response.status_code < 500:
```

Comment:
If I remember correctly, I made the decision to make it show you the message for some particular errors but not others, although I can't remember exactly what the errors were. I could just raise an InternalServerError or Exception in all cases if you prefer.